### ISSUE TITLE
Issues related to Benchmarks

### ISSUE THREAD

**Issue by GiggleLiu** [QuantumBFS/YaoBlocks.jl#53]
* In primitive block benchmark, we should use `put` rather than `repeat`?
* Do not use `rand` in this benchmark, please specify a deterministic position for benchmark.

**Comment by Roger-luo** [comment]
No, the benchmark here is mainly to make sure that each block's apply is dispatched to correct function in certain context. The benchmark of instructions is in YaoArrayRegister. So in principle, we should have both, I just didn't write it in last PR. (I only set up the basic things) 

Then specify a random seed?

**Comment by GiggleLiu** [comment]
Ok, I see. Then about random numbers, why not just specify two numbers to provide better readability?
